products/caregivers/Improve Facility Selection/10-10 Health Apps-10-10CG-[date].md

```diff
@@ -2,71 +2,97 @@
+    - New Facility Confirmation page - avg engagement time (source: [Google Analytics)](https://analytics.google.com/analytics/web/#/analysis/p419143770/edit/8un9wi5nSN-WZ-KqWHHBPA)
 - Backend changes
-    + Does the project introduce any new or unusual infrastructure dependencies?
-    + Do you need to poll any APIs for status?
-    + Are you handling all failure and error cases while in custody of your users's data?
+  - Does the project introduce any new or unusual infrastructure dependencies?
+    - No
+  - Do you need to poll any APIs for status?
+    - No
+  - Are you handling all failure and error cases while in custody of your users's data?
+    - Yes
 - Internal API changes
-    + List new or modified APIs in `vets-api`
-    + Are you deprecating or removing any APIs?
-    + Do you have API documentation?
-    + Describe expected call patterns
+  - List new or modified APIs in `vets-api`
+    - `/v0/caregivers_assistance_claims/facilities` uses the Lighthouse facilities api endpoint
+  - Are you deprecating or removing any APIs?
+    - No
+  - Do you have API documentation?
+    - No, the only change is a new endpoint in the caregivers controller.
+  - Describe expected call patterns
+    - `vets-website` calls this endpoint to a list of paginated facilities
 - External API changes
-    + List new or modified APIs for upstream or external systems
-    + Describe expected call patterns
-    + What PII or PHI will be transmitted to/from the external systems?
+  - List new or modified APIs for upstream or external systems
+    - No Changes
+  - Describe expected call patterns
+    - No Changes
+  - What PII or PHI will be transmitted to/from the external systems?
+    - No Changes
+  - List any required background processing
+    - No Changes
+  - Describe error and dead letter handling
+    - No Changes
 - Background jobs
-    + List any required background processing
-    + Describe error and dead letter handling
+  - List any required background processing
+  - Describe error and dead letter handling
 - Data storage
-    + Describe new or modified databases, tables or columns
-    + Describe indexes and constraints
-    + Identify PII and PHI and where and how it will be stored, processed, expired and deleted
+  - Describe new or modified databases, tables or columns
+    - No Changes. The facility that is ultimately returned is used to pass the facility id as part of the 1010CG Form submission.
+  - Describe indexes and constraints
+    - No Changes
+  - Identify PII and PHI and where and how it will be stored, processed, expired and deleted
+    - No Changes
 - Libraries and dependencies
-    + List new or updated dependencies
+  - List new or updated dependencies
 - Metrics, logging, observability, alerting
-    + Identify key areas to monitor
+  - Identify key areas to monitor
```

Comment:
My thoughts on Metrics, logging etc., Would we identify that we would monitor Facility API for downtimes/latency?

Reply:
Yeah, we'd want to make sure we monitor both the mapbox api and the Lighthouse facilities api. Good callout 👍

Reply:
@hdjustice Where do we usually setup those monitors? I know we have the one that pings the Enrollment System but I'd have to check how we are doing that atm.

Reply:
I think we could use this [dashboard ](https://vagov.ddog-gov.com/dashboard/3vy-h6h-4ek/sitewide-facilities?fromUser=false&refresh_mode=sliding&from_ts=1730924964924&to_ts=1730928564924&live=true)already set up in DD
I can't find anything in DD for mapbox API.  It doesnt come up in any search in dashboards, monitors, services, traces, infrastructure resource catalogue.

Reply:
Monitors are set up in the [Monitors ](https://vagov.ddog-gov.com/monitors/manage?q=1010&order=desc)page.